test/basic.test.js

```diff
@@ -165,4 +165,11 @@ describe("basic", function() {
     css.should.match(/\.a-color/);
     css.should.match(/\.b-color/);
   });
+	it("imports the right file based on relative path", function() {
```

Comment:
NIT: While this changed test looks correct, the one above it looks indented incorrectly...